### ISSUE TITLE
networkx version 2.0 not working with networkx-viewer

### ISSUE THREAD

**Issue by joker234** [jsexauer/networkx_viewer#17]
I wanted to use networkx-viewer after a long time break. Installed a new virtual environment with these versions of the packages:
```
networkx (2.0)
networkx-viewer (0.2.0)
```
If I want to perform a minimal example I got the following error:
```
Traceback (most recent call last):
  File "/home/…/create-network.py", line 33, in <module>
    app = Viewer(G)
  File "/home/…/lib/python3.6/site-packages/networkx_viewer/viewer.py", line 277, in __init__
    EdgeTokenClass=TkPassthroughEdgeToken, **kwargs)
  File "/home/…/lib/python3.6/site-packages/networkx_viewer/viewer.py", line 34, in __init__
    self.canvas = GraphCanvas(graph, width=400, height=400, **kwargs)
  File "/home/…/lib/python3.6/site-packages/networkx_viewer/graph_canvas.py", line 85, in __init__
    self._plot_graph(graph)
  File "/home/…/lib/python3.6/site-packages/networkx_viewer/graph_canvas.py", line 742, in _plot_graph
    layout = self.create_layout(graph, scale=scale, min_distance=50)
  File "/home/…/lib/python3.6/site-packages/networkx_viewer/graph_canvas.py", line 955, in create_layout
    pos= nx.layout._rescale_layout(pos,scale=scale)
AttributeError: module 'networkx.drawing.layout' has no attribute '_rescale_layout'
```
Important part of the source code:
```
G = nx.Graph()

… # adding some nodes and edges

app = Viewer(G)
app.mainloop()
```

I think this error results in the change of `networkx.drawing.layout`, like you can see in the documentation. There is no `_rescale_layout` function anymore. Maybe the `rescale_layout` would do the same for the viewer.
Compare [v1.11](https://networkx.github.io/documentation/networkx-1.11/_modules/networkx/drawing/layout.html) with [v2.0](https://networkx.github.io/documentation/networkx-2.0/_modules/networkx/drawing/layout.html).

Is it possible for you to adjust your awesome package to networkx v2?

**Comment by ShadowAZL** [comment]
Maybe networkx1.x can sovle your problem.  You can alse see [this](https://github.com/ShadowAZL/netdraw/blob/master/netdraw.py). It also can draw dynamic network graph. Just Run it.

**Comment by joker234** [comment]
yes. 1.x would solve this problem, I'm pretty sure. But the interface for networkx changed a bit so I would have to check any networkx function I'm using right now. The linked tool was not working out of the box. (No documentation on how to get it run in less than 30 minutes, too.)

So I'm now stuck at using [gephi](https://gephi.org/), but thanks for your help :)

P.S.: This issue is still valid. It would be awesome, if anyone has the time to fix it, and should only take a few hours.

**Comment by afeser** [comment]
Hi,

I have the same problem, but it seems the repository is not up-to-date. I will solve it if no other solution exists; however, I wonder whether creating a new file for the v2 code or adding `if nx.__version__` statements would give the best result.

Thanks!

**Comment by jsexauer** [comment]
New version released under 0.3.0.  Thanks @afeser for doing the leg work!
